pending/service.go

```diff
@@ -31,6 +39,32 @@ func DefaultService(docSrv documents.Service, repo Repository) Service {
 	}
 }
 
+// Create creates either a new document or next version of an anchored document and stores the document.
+// errors out if there an pending document created already
+func (s service) Create(ctx context.Context, payload documents.UpdatePayload) (documents.Model, error) {
```

Comment:
Do you think we could rename `documents.UpdatePayload` to something like `documents.UpsertPayload` or simply `documents.Payload` ?

Reply:
yes. Lets do it during the cleanup since this is being used by V1 as well?